src/modules/iracing/server/procedures/get-user-summary/index.ts

```
@@ -1,67 +1,54 @@
-      authCode: ctx.iracingAuthCode,
+    const dataLinks = await initialResponse.json();
+    // console.log("Data Links:", dataLinks);
+
+    const linkResponse = await fetch(dataLinks.all.link, {
+      headers: {
+        Authorization: `Bearer ${ctx.iracingAccessToken}`,
+      },
     });
 
-    console.log("Response", res);
+    const data = await linkResponse.json();
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)